src/java/org/apache/cassandra/repair/autorepair/FixedSplitTokenRangeSplitter.java

```diff
@@ -49,10 +49,10 @@ public class FixedSplitTokenRangeSplitter implements IAutoRepairTokenRangeSplitt
      * Selecting the default value is tricky. If we select a small number, individual repairs would be heavy.
      * On the other hand, if we select a large number, too many repair sessions would be created.
      * <p>
-     * The default value of 64 works well for non-vnodes, but with vnodes, it can result in too many repair sessions
-     * because the total number of repair sessions on a node is calculated as:
+     * The default value of 64 works well for non-vnodes, but with vnodes, it might result in too many repair sessions
+     * and to cap the number for vnodes, we use the following formula:
```

Comment:
I think we should tweak this just a little but as I think we don't need to worry about too many repair sessions for vnodes anymore since we subdivide.  We also reference a default of 64 here where the default is 32.

Think would be nice to right a more concise version of what is written here:

https://github.com/jaydeepkumar1984/cassandra/pull/47/files#r1989601197

Maybe can just simplify to:

```
If vnodes are configured using <code>num_tokens</code>, attempts to evenly subdivide subranges by each range using the following formula:

<retain formula on lines 54-56>
```

Reply:
Sure, just updated as per the comment from https://github.com/jaydeepkumar1984/cassandra/pull/47/files#r1989601197